src/components/Votes.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { updateArticle } from "../api";
+
+const Votes = ({ article_id, votes }) => {
+  const [optimisticVotes, setOptimisticVotes] = useState(0);
+
+  const incrementVotes = () => {
+    setOptimisticVotes((currVotes) => {
+      return currVotes + 1;
+    });
+    updateArticle(article_id).catch(() => {
+      alert("Something went wrong. Try another time");

```

Comment:
i would advise against using the `alert` function in FE

It's a blocking piece of code and isn't the best user experience

Showing a message or just decreasing the number (like you already are) should be enough